src/ui/common/src/utils/workflows.tsx

```diff
@@ -135,14 +141,21 @@ export type GetWorkflowResponse = {
   workflow_dag_results: WorkflowDagResultSummary[];
 };
 
+export function getSavedObjectIdentifier(savedObject: SavedObject): string {
+  if ('format' in savedObject.spec.parameters) {
```

Comment:
I'm not sure if we have a recommended way of switching on a type? I tried the `as` syntax to check the type but it didn't do what I wanted.

Reply:
Is there a `ServiceType` field we can use anywhere? That should be safer than examining fields. @agiron123 should know better on doing the casting part properly.

Reply:
Yeah that seems better - I'll check check `ServiceType` in the conditionals.